src/lambda/handler-runner/docker-runner/DockerImage.js

```diff
@@ -1,31 +1,45 @@
-import execa from 'execa'
+// import execa from 'execa'
 import promiseMemoize from 'p-memoize'
 import debugLog from '../../../debugLog.js'
+import serverlessLog from '../../../serverlessLog.js'
 
 export default class DockerImage {
   #imageNameTag = null
+  #dockerClient = null
 
-  constructor(imageNameTag) {
+  constructor(imageNameTag, dockerClient) {
     this.#imageNameTag = imageNameTag
+    this.#dockerClient = dockerClient
   }
 
-  static async _pullImage(imageNameTag) {
+  static async _pullImage(imageNameTag, dockerClient) {
     debugLog(`Downloading base Docker image... (${imageNameTag})`)
 
     try {
-      await execa('docker', [
-        'pull',
-        '--disable-content-trust=false',
-        imageNameTag,
-      ])
+      const stream = await dockerClient.pull(imageNameTag, {
+        disableContentTrust: false,
+      })
+
+      dockerClient.modem.followProgress(
+        stream,
+        (err) => {
+          if (err) {
+            throw err
+          }
+        },
+        (event) => {
+          const { status, id } = event
+          serverlessLog(`${status}${id ? ` - ${id}` : ''}`)
+        },
+      )
```

Comment:
`_pullImage()` does not seems to work (This is the cause of CI failure). I think this part should be promise and awaited.

```suggestion
      await new Promise((resolve, reject) => {
        dockerClient.modem.followProgress(
          stream,
          (err) => {
            if (err) {
              reject(err)
            }
            resolve()
          },
          (event) => {
            const { status, id } = event
            serverlessLog(`${status}${id ? ` - ${id}` : ''}`)
          },
        )
      })
```